### PR TITLE
PulseAudio: fall back to default device if chosen device is no longer present.

### PR DESCRIPTION
**Pull request by mkrautz**

### CODE REVIEW

File: src/mumble/PulseAudio.cpp [hunk 1]
```diff
@@ -153,6 +153,28 @@ PulseAudioSystem::~PulseAudioSystem() {
 	pa_threaded_mainloop_free(pam);
 }
 
+QString PulseAudioSystem::outputDevice() const {
+	QString odev = g.s.qsPulseAudioOutput;
+	if (odev.isEmpty()) {
+		odev = qsDefaultOutput;
+	}
+	if (!qhOutput.contains(odev)) {
```
Comment (hacst) [inline src/mumble/PulseAudio.cpp]: I assume this actually needs contains and not equals? In what way will they differ yet still match?

**Comment by hacst** [comment]
See comment. LGTM.

File: src/mumble/PulseAudio.cpp (hunk 1 above)
Comment (mkrautz) [inline src/mumble/PulseAudio.cpp]: qhOutput is a QHash

**Comment by mkrautz** [comment]
Landed as a50a120b2c398afe41801ae7d14c805ceac73110.

**Pull request closed without merging**